features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Login/RedirectToLogin.jsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import Configurations from 'Config';
+import { FormattedMessage } from 'react-intl';
+
+const page = Configurations.app.context + '/services/configs';
+
+/**
+ *
+ * Just doing the redirection, If you want to trigger redirection to login page , Import this util method and use.
```

Comment:
**login page , Import** should be **login page, import**

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/8152